src/System.Runtime.Extensions/tests/Performance/System/Perf.Convert.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System
+{
+    public class Perf_Convert
+    {
+        private byte[] InitializeBinaryDataCollection(int size)
+        {
+            var random = new Random();
```

Comment:
I doubt it'll make a difference in this case, but it'd be good to pass a fixed seed to Random so that the data is the same on every run.